app/services/course/assessment/question/programming_import_service.rb

```diff
@@ -89,7 +89,21 @@ def build_template_file_records(template_files)
     end
   end
 
-  # Builds the test case records from the test report.
+  # Goes through each test report file and combines all the test cases contained in them.
+  #
+  # @param [Hash<String, String>] test_reports The test reports from evaluating the package.
```

Comment:
Can document the detailed formats of the test_reports?

Reply:
Done, it's on the following few lines.